src/test/java/io/github/sashirestela/openai/SimpleOpenAIAnyscaleTest.java

```diff
@@ -0,0 +1,28 @@
+package io.github.sashirestela.openai;
+
+import static io.github.sashirestela.openai.SimpleOpenAIAnyscale.AUTHORIZATION_HEADER;
+import static io.github.sashirestela.openai.SimpleOpenAIAnyscale.BEARER_AUTHORIZATION;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.net.http.HttpClient;
+import org.junit.jupiter.api.Test;
+
+class SimpleOpenAIAnyscaleTest {
+    @Test
+    void shouldPrepareBaseOpenSimpleAIArgsCorrectly() {
```

Comment:
You could check that the default url is taken when you no pass any one and the same for the httpClient